### DISCUSSION TITLE
Widgets/History/Comments/Updates

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17401] [How to]
Hello,

Something wrong with this Widgets? Previously in version 6.4 you could see History/Comments/Updates, now after updating to 6.5 the Comments and Updates that occur day by day are not displayed
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/57bb1f28-bc99-460a-bdee-bcc9f4a08a62)

**Comment by krseguros** [comment]
Greetings,
Can someone tell me why this happens? check the version https://gitstable.yetiforce.com/index.php and everything is fine
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/06e874a4-f5c0-4db5-af63-e577fbcf4140)
In my instance this widget is not showing the updates, I have the latest version 6.5

**Reply by vovpff** [reply]
Maybe u need to activate module tracking?

**Comment by krseguros** [comment]
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/9ae202bb-7308-403a-89fb-e1915d700262)

If this refers to you, if it is active, I do not have any restrictions on users. Before updating, the changes and updates were displayed fine, now after updating to 6.5 it is not displayed.
